features/smartanswers.feature

```diff
@@ -29,13 +29,12 @@ Feature: Smart Answers
 
   Scenario: Check stepping through a smart answer that uses Imminence
     Then I should be able to visit:
-    | Path                                              |
-    | /landlord-immigration-check                       |
-    | /landlord-immigration-check/y                     |
-    | /landlord-immigration-check/y/SW1A2AA             |
-    | /landlord-immigration-check/y/SW1A2AA/yes         |
-    | /landlord-immigration-check/y/SW1A2AA/yes/yes     |
-    | /landlord-immigration-check/y/SW1A2AA/yes/yes/yes |
+    | Path                                                                             |
+    | /benefit-cap-calculator                                                          |
+    | /benefit-cap-calculator/y                                                        |
+    | /benefit-cap-calculator/y/yes/no/none/child_benefit/100.0/400.0/single/SW1A2AA   |
+    | /benefit-cap-calculator/y/yes/no/none/child_benefit/100.0/400.0/couple/SW1A2AA   |
+    | /benefit-cap-calculator/y/yes/no/none/widow_pension/400.0/100.0/single/SW1A2AA   |
```

Comment:
Is there a particular logic for needing all these URLs? i.e. do they test different imminence features? I'm sort of wondering why we need these 5 URLS would say just the third meet the test?

If there's some reason to it, it'd be good to have it in the commit message.

By the way there's some other tests too: https://github.com/alphagov/smokey/blob/2398362070b4798a77c6c4c2e89d6c39d953d426/features/smartanswers.feature#L107-L108